pkgs/native_toolchain_c/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.3.1
+
+- Added MSVC arm64 toolchain for windows-arm64 in flutter.
```

Comment:
Remove Flutter. Maybe people want to use it with standalone Dart arm64 on Windows as well. =)

Reply:
You're right, my view was too narrow.